app/jobs.js

```diff
@@ -99,4 +99,23 @@ const jobs = (jenkins, db) => {
   };
 };
 
+function dumpRequesError(r) {
```

Comment:
Use the new ES6 syntax for functions:

```
const dumpRequestError = (r) => {
  ... 
}
```

Reply:
Will fix. But what is the benefit of this notation?
These are internal functions, so I don't expect them to be used externally.

Reply:
There are several advantages to use this notation. The first one is that the `this` variable is properly set inside them (scope). They are not focused on exporting :)